src/once-ui/components/InteractiveDetails.tsx

```
@@ -27,7 +27,14 @@ const InteractiveDetails: React.FC<InteractiveDetailsProps> = forwardRef<
           {label}
         </Text>
         {iconButtonProps?.tooltip && (
-          <div onClick={(e) => e.stopPropagation()}>
+          <div
+            onClick={(e) => e.stopPropagation()}
+            onKeyDown={(e) => {
+              if (e.key === "Enter" || e.key === " ") {
+                e.stopPropagation();
+              }
+            }}
+          >
```

Biome (2.4.10)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)